Stacks/ValidateParaentheses.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections;
+
+ class ValidatingParaentheses{
+  public static void Main(string[] args){
+    Console.WriteLine(validate(""));
+  }
+
+  private static bool validate(string str){
+    Stack s = new Stack();
```

Comment:
It is a good idea to include what type of Stack it is, is this a stack of integers or characters or something else?